### ISSUE TITLE
Fatal Error: Cannot recover after last error. Any further errors will be ignored.

### ISSUE THREAD

**Issue by Arttse** [validator/validator#350]
Hello! Thank you for your work on the `validator`!
Since nobody answers in [Gitter](https://gitter.im/validator/validator), I write my question here.

I ran into this error:
```
Fatal Error: Cannot recover after last error. Any further errors will be ignored.
```

I created one example. You can see [here](https://gist.github.com/Arttse/2bcb366a3d645d4a507f7197dc3de03f)

I want to see all errors on the page. How can I do it? There is some option for standalone web server?
Probably this is a bug? I see if check on https://validator.w3.org/nu by `text input` and then click «Check» button the code is clipped in the `textarea`

Thanks!

**Comment by sideshowbarker** [comment]
The short answer is that you must first fix whatever markup error the checker is reporting that fatal error for. After you fix that error, you then need to re-check the document and the checker will then report any subsequent errors.

The longer answer is that usually when you see that error it means the checker has run into a markup error condition that an HTML parser would need to do backtracking for in order to do error recovery/fixup so that it can then continue parsing.

A typical case like that is if you have one `a` element nested somewhere inside another `a` element.

The non-streaming HTML parsers used in browsers are able to do error recovery/fixup from such conditions, but the streaming HTML parser that we use in the checker cannot. Basically, because it does streaming parsing, it cannot do the backtracking necessary to fixup/recover from that error.
